agenda/presentation/src/main/java/com/gkp/agenda/presentation/util/Util.kt

```diff
@@ -0,0 +1,46 @@
+package com.gkp.agenda.presentation.util
+
+import java.time.LocalDate
+private const val CURRENT_AND_NEXT_SIX_DATES = 7
+
+fun getFullNameInitials(fullName: String): String {
+    val names = fullName.split(" ")
+    val initialsBuilder = StringBuilder()
+    if (names.size == 1) {
+        initialsBuilder.append(names[0].take(2).uppercase())
+        return initialsBuilder.toString()
+    }
+    if (names.size == 2) {
+        initialsBuilder.append("${names[0].first()}${names[1].first()}".uppercase())
+        return initialsBuilder.toString()
+    }
+    if (names.size >= 3) {
+        initialsBuilder.append("${names[0].first()}${names[2].first()}".uppercase())
+        return initialsBuilder.toString()
+    }
+    return ""
+}
+
+
+fun getCurrentAndNextSixWeekDayDates(
+    selectedDate: LocalDate
+): List<DateWithSelected> {
+    val today = LocalDate.now()
+
+    return buildList {
+        repeat(CURRENT_AND_NEXT_SIX_DATES) {
+            val date = today.plusDays(it.toLong())
+            add(
+                DateWithSelected(
+                    date = date,
+                    isSelected = date == selectedDate
+                )
+            )
+        }
+    }
+}
+
+data class DateWithSelected(
+    val date: LocalDate,
+    val isSelected: Boolean,
+)
```

Comment:
While your preference how you create a list is completely up to you, just to share mine:
```
val today = LocalDate.now()
return (0 until CURRENT_AND_NEXT_SIX_DATES).map {
    DateWithSelected(
        date = today.plusDays(it.toLong()),
        isSelected == date == selectedDate
    )
}
```
Has one less indentation 😄